spec/beaker-hostgenerator/generator_spec.rb

```diff
@@ -98,12 +57,20 @@ module BeakerHostGenerator
             fixture_hash['environment_variables'].each do |key, value|
               ENV[key] = value
             end
+
             cli = BeakerHostGenerator::CLI.new(arguments)
-            test_hash = YAML.load(cli.execute)
+            if fixture_hash['expected_exception']
+              # Turn fully-qualified classname string into an actual Class object.
+              expected_class = eval fixture_hash['expected_exception']
```

Comment:
> Eval is a pretty loaded gun. 

If we wanted to get rid of it the alternate implementation I provided above works fine as well.
`const_get` just doesn't know how to recurse into the each module, so you have to hold it's hand.